### PR TITLE
Remove inverted-colors UA styles to match spec

### PR DESCRIPTION
**Pull request by lukewarlow**
#### 05dae4e20ce5190d8a954443dbe6a0b5df3c058e
<pre>
Remove inverted-colors UA styles to match spec
<a href="https://bugs.webkit.org/show_bug.cgi?id=267776">https://bugs.webkit.org/show_bug.cgi?id=267776</a>

Reviewed by NOBODY (OOPS!).

The CSSWG resolved to remove these UA styles from the spec.

See decision: <a href="https://github.com/w3c/csswg-drafts/issues/9674#issuecomment-1896256196">https://github.com/w3c/csswg-drafts/issues/9674#issuecomment-1896256196</a>

* LayoutTests/fast/media/mq-inverted-colors-ua-styles-expected.txt: Removed.
* LayoutTests/fast/media/mq-inverted-colors-ua-styles-img-expected.html: Removed.
* LayoutTests/fast/media/mq-inverted-colors-ua-styles-img.html: Removed.
* LayoutTests/fast/media/mq-inverted-colors-ua-styles.html: Removed.
* Source/WebCore/css/html.css:
(@media (inverted-colors) img:not(picture&gt;img), picture, video): Deleted.
</pre>
https://github.com/WebKit/WebKit/commit/05dae4e20ce5190d8a954443dbe6a0b5df3c058e

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/34811 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/13635 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/36823 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/37499 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/31415 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/35954 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/16044 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/10721 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/30369 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/35339 "Passed tests") | [❌ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/11558 "Found 1 new test failure: accessibility/smart-invert.html (failure)") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/31003 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/10093 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/10175 "Passed tests") | [❌ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/31079 "Found 1 new test failure: imported/w3c/web-platform-tests/resize-observer/callback-cross-realm-report-exception.html (failure)") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/38765 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/31617 "Passed tests") | [❌ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/31406 "Found 1 new test failure: accessibility/smart-invert.html (failure)") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/36211 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/10272 "Built successfully") | [❌ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/50/builds/8184 "Found 1 new test failure: accessibility/smart-invert.html (failure)") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/34178 "Passed tests") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/12101 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/10818 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/11167 "Built successfully") | | |

### CODE REVIEW

**Comment by lukewarlow** [comment]
media-controls.css contains:

```css
/* Default support for "Smart Invert" where all content color except media is inverted. */ 
@media (inverted-colors) {
    :host { filter: invert(100%); } /* WebKit native audio and video. */
    picture { filter: none; } /* Don't invert the control buttons. */
}
```

Idk what would be preferred here should I remove that style too?

File: Source/WebCore/css/html.css [hunk 1]
```diff
@@ -1453,12 +1453,6 @@ applet, embed, object, img {
     }
 }
 
-/* Default support for "Smart Invert" where all content color except media is inverted. */
-/* https://drafts.csswg.org/mediaqueries-5/#inverted */
-@media (inverted-colors) {
-    img:not(picture>img), picture, video { filter: invert(100%); } /* Images and videos double-inverted. */
-}
-
```
Comment (cookiecrook) [inline Source/WebCore/css/html.css:1461]: Removing this would break support for a major accessibility feature on multiple platforms.

**Review by cookiecrook — CHANGES_REQUESTED** [review]

File: Source/WebCore/css/html.css (hunk 1 above)
Comment (lukewarlow) [reply Source/WebCore/css/html.css:1461]: Would you be able to share this concern on https://github.com/w3c/csswg-drafts/issues/9674 where the CSSWG resolved to move this UA style to an author suggestion?

**Comment by lukewarlow** [comment]
Marking as draft for now, depending on end situation I'll either close (most likely) or mark as ready.

Thanks for the quick review btw @cookiecrook

**Comment by lukewarlow** [comment]
Gonna go ahead and close this for now, can reopen if the CSSWG *and* WebKit agree it's the best course of action.

**Pull request closed without merging**